### DISCUSSION TITLE
阿里云盘 open出现未知问题

### DISCUSSION THREAD

**Discussion by Rpower666** [AlistGo/alist#3680] [Q&A]
alist v3.12.2，failed get objs: failed to list objs: Post "https://api.nn.ci/alist/ali_open/token": EOF，各位有遇到这个问题的吗，昨天升级使用的时候还好好的

**Comment by Kaneshiro1998** [comment]
顶一下，我也是这个问题

**Comment by Rpower666** [comment]
Kaneshiro1998 ***@***.***>于2023年3月4日 周六12:45写道：

> 顶一下，我也是这个问题
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/3680#discussioncomment-5199487>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APIAIYKR5DJRGUDQHN5IISLW2LCF3ANCNFSM6AAAAAAVNDRRFE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
已解决，主要是还是服务器网络问题导致的

**Comment by qinhaiming** [comment]
还是不行

**Reply by qinhaiming** [reply]
Failed init storage: Post "https://api.nn.ci/alist/ali_open/token": EOF

**Comment by muzhiye** [comment]
我也是这问题解决了吗？

**Reply by Kaneshiro1998** [reply]
我直接用回阿里云盘的那个接口可以了。。。虽然写着不再维护

**Comment by Ezio125** [comment]
我这两天也是出现这个问题，非要重新获取token当天就又行了，但是隔天又会出现这个。很烦躁。

**Comment by Kaneshiro1998** [comment]
不用阿里云盘Open，回去用阿里云盘的那个接口就好了，也是无语

**Comment by Ezio125** [comment]
已经解决了，用直连模式。

**Reply by sjqboss** [reply]
> 已经解决了，用直连模式。

什么是直连模式

**Comment by flockitcom** [comment]
如题，如果遇到类似的问题，可以通过以下方式解决：

修改驱动设置中的Oauth令牌链接，将其替换为https://api-cf.nn.ci/alist/ali_open/token
